src/ports/postgres/modules/convex/mlp.sql_in

```diff
@@ -1474,13 +1480,15 @@ CREATE AGGREGATE MADLIB_SCHEMA.mlp_minibatch_step(
         /* warm_start_coeff */    DOUBLE PRECISION[],
         /* lambda */              DOUBLE PRECISION,
         /* batch_size */          INTEGER,
-        /* n_epochs */            INTEGER
+        /* n_epochs */            INTEGER,
+        /* momentum */            DOUBLE PRECISION,
+        /* is_nesterov */         BOOLEAN
         )(
     STYPE=DOUBLE PRECISION[],
     SFUNC=MADLIB_SCHEMA.mlp_minibatch_transition,
     m4_ifdef(`__POSTGRESQL__', `', `prefunc=MADLIB_SCHEMA.mlp_minibatch_merge,')
     FINALFUNC=MADLIB_SCHEMA.mlp_minibatch_final,
-    INITCOND='{0,0,0,0,0,0,0,0,0,0,0,0}'
+    INITCOND='{0,0,0,0,0,0,0,0,0,0,0,0,0,0}'
 );
 -------------------------------------------------------------------------
 
```

Comment:
Can you please update the user docs with momentum and nesterov related optimizer params? Is it also a good time to update the MLP design doc?

Reply:
+1 will do